packages/fuse/fuse.spec

```diff
@@ -0,0 +1,93 @@
+Name:		%{_cross_os}fuse
+Version:	2.9.9
+Release:	1%{?dist}
+
+Summary:	File System in Userspace (FUSE) v2 utilities
+License:	GPL-1.0-or-later
+URL:		http://fuse.sf.net
+Source0:	https://github.com/libfuse/libfuse/releases/download/fuse-2.9.9/fuse-2.9.9.tar.gz
+
+Patch1: fuse2-0001-More-parentheses.patch
+# https://bugzilla.redhat.com/show_bug.cgi?id=970768
+Patch2: fuse2-0002-add-fix-for-namespace-conflict-in-fuse_kernel.h.patch
+# https://github.com/libfuse/libfuse/commit/4f8f034a8969a48f210bf00be78a67cfb6964c72
+# backported for fuse2
+Patch3: fuse2-0003-make-buffer-size-match-kernel-max-transfer-size.patch
+# https://bugzilla.redhat.com/1694552#c7
+# https://github.com/libfuse/libfuse/pull/392
+# backported for fuse2
+Patch4: fuse2-0004-Whitelist-SMB2-found-on-some-NAS-devices.patch
+# cherry-picked from upstream
+Patch5:	fuse2-0005-Whitelist-UFSD-backport-to-2.9-branch-452.patch
+# cherry-picked from upstream
+Patch6: fuse2-0006-Correct-errno-comparison-571.patch
+# cherry-picked from upstream
+# https://bugzilla.redhat.com/show_bug.cgi?id=1984776
+Patch7: fuse2-0007-util-ulockmgr_server.c-conditionally-define-closefro.patch
+
+Conflicts:	filesystem < 3
+BuildRequires:	libselinux-devel
+BuildRequires:	autoconf, automake, libtool, gettext-devel, make
+
+%description
+With FUSE it is possible to implement a fully functional filesystem in a
+userspace program. This package contains the FUSE v2 userspace tools to
+mount a FUSE filesystem.
+
+%package libs
+Summary:	File System in Userspace (FUSE) v2 libraries
+License:	LGPL-2.1-or-later
+Conflicts:	filesystem < 3
+
+%description libs
+Devel With FUSE it is possible to implement a fully functional filesystem in a
+userspace program. This package contains the FUSE v2 libraries.
+
+%prep
+%autosetup -n fuse-2.9.9 -p 1
+
+# ./makeconf.sh
```

Comment:
If we do use v2.9.9, we should change how to compile the package. I've been playing with it and I think we should do something like this:

```
./makeconf.sh
./configure --prefix=%{_cross_prefix} #... and all other flags
```

And disable all the things we don't want like `--disable-utils` and `--disable-static`. Then, since you used the correct prefixes, you could just `%make_install` in the install section.

Reply:
soci-snapshotter's main dependency is the fusermount executable from util, so we need --enable-utils. The only thing that soci-snapshotter does (directly) is query whether there is an executable fusermount on its path; it looks at the error (if any) from the query and throws away the information about the executable itself. I have not looked into all of the dependencies to see if they, then, do anything with fusermount.